docs/source/deploypeer/peerdeploy.md

```diff
@@ -0,0 +1,111 @@
+    ```
+    ├── organizations
+      └── peerOrganizations
+          └── org1.example.com
+              ├── msp
+              └── peers
+                  └── peer0.org1.example.com
+                      ├── msp
+                      └── tls
+    ```
+2. While **cryptogen** is a convenient utility that can be used to generate certificates for a test network, it should **never** be used on a production network. The core requirement for certificates for Fabric nodes is that they are Elliptic Curve (EC) certificates. You can use any tool you prefer to issue these certificates (for example, OpenSSL). However, the Fabric CA streamlines the process because it generates the Membership Service Providers (MSPs) for you. Regardless of which certificate provider you choose, you should have already used it to generate the peer enrollment certificate, TLS certificate, private keys, and the MSPs that Fabric must consume. Refer to the [CA deployment](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/cadeploy.html) and [Registering and enrolling identities with a CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html) topics for instructions on how to generate these certificates. You need to generate the following sets of certificates:
+    - **Peer TLS certificates**
+    - **Peer organization admin certificates (organization MSP)**
+    - **Peer enrollment certificates (peer local MSP)**  
+3. You will either need to use the Fabric CA client to generate the certificates directly into the recommended folder structure or you will need to copy the generated certificates to their recommended folders after they are generated. Whichever method you choose, must users are ultimately likely to script this process so it can be repeatable as needed. A list of the certificates and their locations is provided here for your convenience.
+      - **Peer TLS certificate and private key**:
+        - Copy the TLS CA TLS certificate `tls-cert.pem` to `organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/tls-cert.pem`. The path and name of the certificate corresponds to `peer.tls.rootcert.file` parameter in `core.yaml`.
+        - Copy the TLS CA root certificate `ca-cert.pem` to `organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/ca-cert.pem`. The path and name of the certificate corresponds to `peer.tls.clientRootCAs.files` parameter in `core.yaml`. Only required if mutual authentication (`clientAuthRequired`) is set to true.
+        - After you have generated the peer TLS certificate, rename the generated private key in the `keystore` folder to `peer0-key.pem` so that it can more easily be recognized later on as being a private key.
```

Comment:
```suggestion
        - After you have generated the peer TLS certificate, the certificate will have been generated in the `signcerts` directory, and the private key will have been generated in the `keystore` directory. Rename the generated private key in the `keystore` folder to `peer0-key.pem` so that it can more easily be recognized later on as being a private key.
```